alignak/objects/config.py

```diff
@@ -2234,12 +2256,24 @@ def add_error(self, txt):
         self.conf_is_correct = False
 
     def show_errors(self):
-        """Loop over configuration_errors and log them
+        """
+        Loop over configuration warnings and log them as INFO log
+        Loop over configuration errors and log them as INFO log
+
+        Note that the warnings and errors are logged on the fly during the configuration parsing.
+        It is not necessary to log as WARNING and ERROR in this function which is used as a sum-up
+        on the end of configuration parsing when an error has been detected.
 
         :return:  None
         """
-        for err in self.configuration_errors:
-            logger.error(err)
+        if self.configuration_warnings:
+            logger.info("Configuration warnings:")
+            for msg in self.configuration_warnings:
+                logger.info(msg)
```

Comment:
No, bad idea. As explained in the comment, using INFO is better beceause these logs are simply used to build a sum-up when configuration check is finished. If we use WARNING and ERROR, each warning and error will appera twice in the logs ... bad idea if we want to parse the logs with an application like LogStash :/